dragonfly_schema/_openapi.py

```diff
@@ -2,14 +2,20 @@
 from pydantic.schema import schema
 from typing import Dict
 
+VERSION = None
+
+try:
+    VERSION = '.'.join(get_distribution('honeybee_schema').version.split('.')[:3]),
```

Comment:
This should be `'dragonfly_schema'` and not `'honeybee_schema'`. Once this is fixed, we should be able to merge.

Reply:
Ah yes good catch... On it.